component/main.jsonnet

```diff
@@ -7,6 +7,46 @@ local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.csi_driver_smb;
 
+local isOpenshift = std.startsWith(inv.parameters.facts.distribution, 'openshift');
+
+local namespace =
+  kube.Namespace(params.namespace)
+  {
+    metadata+: {
+      annotations+: {
+        // Allow Pods to be scheduled on any Node
+        [if isOpenshift then 'openshift.io/node-selector']: '',
+      },
+    },
+  };
+
+local rolebinding = if isOpenshift then
```

Comment:
Duplicated `isOpenshift` check (here and at the bottom)

Reply:
I just seems like code duplication to me (and one of the occurrences will probably be overlooked the next time this gets touched). I would prefer to have the check only in one place, but then again it's not a blocker for me.